libs/components/popovers/src/lib/modules/popover/popover.component.ts

```diff
@@ -231,6 +231,14 @@ export class SkyPopoverComponent implements OnDestroy {
     this.#isMarkedForCloseOnMouseLeave = true;
   }
 
+  /**
+   * Whether the popover body has interactable content
```

Comment:
This is a component, so we don't need JSDocs (or `@internal`) for methods/properties (we only do that for services).

Reply:
Also, the method name is self-explanatory, so we can just remove the comments. Good forward thinking, though. 👍